app/src/types/index.ts

```diff
@@ -20,17 +20,23 @@ export type CartItem =
 export type EtherscanGroup = 'tx' | 'token' | 'address';
 
 // Cart prediction entries
-export type CartPrediction = { matching: number; matchingToken: TokenInfo };
+export type CartPrediction = { matching: number | boolean; matchingToken: TokenInfo };
 export type CartPredictions = {
   [grantId: string]: CartPrediction[];
 };
 
+// Version/name given to the default localForage instance
+export type LocalForageConfig = {
+  name: string;
+  version: number;
+};
+
 // Generic localStorage type to store key->any
-export type LocalStorageAnyObj = {
+export type LocalForageAnyObj = {
   [key: string]: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 };
-export type LocalStorageData = {
+export type LocalForageData = {
   ts?: BigNumberish;
```

Comment:
What is `ts`? I'd suggest a more descriptive variable name or a comment

Reply:
I've removed `ts` -- we was using it to throttle refreshes on the cache (by time) but everything uses the blockNumber now 👍